src/bindings/overlay.js

```diff
@@ -37,6 +37,15 @@ const ALLOWED_ATTRIBUTES = {
   }
 };
 
+const DOM_NAMESPACES = {
+  'html': 'http://www.w3.org/1999/xhtml',
+  'xul': 'http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul',
+
+  // Reverse map for overlays.
+  'http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul': 'xul',
+  'http://www.w3.org/1999/xhtml': 'html',
```

Comment:
The reverse map shouldn't be required if you pass actual DOM Attribute objects or `{ name, namespaceURI }` objects to `toAttrAllowed`.